src/pages/bubblegum-v2/transfer-cnfts.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Transferring Compressed NFTs
+metaTitle: Transferring Compressed NFTs | Bubblegum v2
+description: Learn how to transfer compressed NFTs on Bubblegum
+---
+
+The **transferV2** instruction can be used to transfer a Compressed NFT from one owner to another. To authorize the transfer, either the current owner or the delegate authority — if any — must sign the transaction.
+
+Note that this instruction updates the Compressed NFT and therefore replaces the leaf on the Bubblegum Tree. This means additional parameters must be provided to verify the integrity of the Compressed NFT. Since these parameters are common to all instructions that mutate leaves, they are documented [in the following FAQ](/bubblegum-v2/faq#replace-leaf-instruction-arguments). Fortunately, we can use a helper method that will automatically fetch these parameters for us using the Metaplex DAS API.
+
+{% callout title="Transaction size" type="note" %}
+If you encounter transaction size errors, consider using `{ truncateCanopy: true }` with `getAssetWithProof`. See the [FAQ](/bubblegum-v2/faq#replace-leaf-instruction-arguments) for details.
+{% /callout %}
+
+## Transfer a Bubblegum V2 Compressed NFT
+
+The instruction accepts the following parameters:
+
+- **Leaf Owner**: The current owner of the Compressed NFT
```

Comment:
Note also the collection `permanentTransferDelegate` can transfer

Reply:
Nit: I would backtick permanent transfer delegate in this section to be consistent with other sections